src/tree/localProject/LocalProjectTreeItem.ts

```diff
@@ -24,7 +24,7 @@ export class LocalProjectTreeItem extends AzExtParentTreeItem {
     }
 
     public get iconPath(): TreeItemIconPath {
-        return treeUtils.getThemedIconPath('CreateNewProject');
+        return treeUtils.getThemedIconPath('folder');
```

Comment:
Uh we're not usually in the business of creating our own icons. You can use any existing icon you find, but otherwise I think you should talk to an actual icon person if you need a new one (Jing probably knows who to talk to).

Reply:
The consensus in standup on Friday was that if you and Nathan are OK with it we can use this one otherwise we can escalate it to a designer

Reply:
My concern is not the icon itself (that's very subjective), but how we got it. I think it's a slippery slope if we start designing our own icons and I think there should be plenty of existing icons to choose from. For example if you want a folder without the lightning bolt, you could go with this one: https://github.com/microsoft/vscode-azuretools/blob/main/appservice/resources/dark/folder.svg

There's probably some folder icons in the Azure icon set, too

Reply:
Also on a somewhat related note, we want to start using `ThemeIcon` as much as possible so that users can use their own icons and get it from the theme instead of us. We can do that for simple things like a folder, but we'll run into problems for Azure-specific icons.

https://code.visualstudio.com/updates/v1_54#_icon-contribution-point

Reply:
Going with the folder codicon sounds best. For whatever reason that specific icon doesn't seem to work with `ThemeIcon` right now though lol. So pending on https://github.com/microsoft/vscode/issues/119028